fieldservice_vehicle/models/fsm_order.py

```diff
@@ -15,7 +15,7 @@ def _get_default_vehicle(self):
 
     @api.model
     def create(self, vals):
-        res = super().create(vals)
-        if not res.vehicle_id and res.person_id:
-            res.vehicle_id = res.person_id.vehicle_id.id or False
-        return res
+        if not vals.get('vehicle_id', False) and vals.get('person_id', False):
```

Comment:
The False second argument is redundant
```suggestion
        if not vals.get('vehicle_id') and vals.get('person_id'):
```